editors/shared/components/content-card.tsx

```diff
@@ -4,22 +4,26 @@ interface AtlasCardProps extends PropsWithChildren {
   tagText?: string;
   variant?: "gray" | "green" | "blue";
   className?: string;
+  isShowTag?: boolean;
 }
 
 const ContentCard = ({
   tagText = "Official Atlas",
   variant = "gray",
   children,
   className,
+  isShowTag = false,
```

Comment:
the variable name has two different verbs (is and show), it should be `showTag`

Reply:
you are right, done